server/controllers/dashboard/debtorGroups.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const moment = require('moment');
+
+const report = require('../finance/reports/agedDebtors');
+
+// time until the cached value expires - currently set at 5 minutes
+const CACHE_TTL = 300000;
```

Comment:
This is cool!  It could be way more generic.  Do you think that we should be using variables local to a module as the cache?  It seems rather amateur, but I cannot think of a better way at the moment...